### PR TITLE
build: don't depend on npm packages from public api tests

### PR DESCRIPTION
**Pull request by IgorMinar**
we need only the inputs from ng_module.

the missing piece is to ensure that we depend on all secondary entry points

### CODE REVIEW

**Comment by IgorMinar** [comment]
cc: @petebacondarwin

File: tools/ts-api-guardian/lib/cli.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export function startCli() {
         // Under bazel, give instructions how to use bazel run to accept the golden file.
         if (!!process.env['BAZEL_TARGET']) {
           console.error('\n\nAccept the new golden file:');
-          console.error(`  bazel run ${process.env['BAZEL_TARGET']}.accept`);
+          console.error(`  bazel run ${process.env['BAZEL_TARGET'].replace(/_bin$/, "")}.accept`);
```
Comment (petebacondarwin) [inline tools/ts-api-guardian/lib/cli.ts]: 💯

File: tools/public_api_guard/public_api_guard.bzl [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# found in the LICENSE file at https://angular.io/license
+load("//tools/ts-api-guardian:index.bzl", "ts_api_guardian_test")
+
+def generate_targets(golden_files):
+    for golden_file in golden_files:
+        entry_point = golden_file[:-len(".d.ts")]
+        [package_name, entry_point_tail] = entry_point.split("/", 1)
+        directory_name = entry_point_tail.split("/")[-1]
+        target_suffix = "/" + entry_point_tail if package_name != entry_point_tail else ""
+        actual_file = "packages/%s/%s/%s.d.ts" % (package_name, target_suffix, directory_name)
```
Comment (petebacondarwin) [inline tools/public_api_guard/public_api_guard.bzl]: if `target_suffix` is `""` then won't we end up with `actual_file` looking like `packages/common//testing.d.ts` (note the double slash)?

Comment (petebacondarwin) [reply tools/public_api_guard/public_api_guard.bzl]: Actually if the `target_suffix` is not empty then it must start with a `/` right?

Comment (petebacondarwin) [reply tools/public_api_guard/public_api_guard.bzl]: So you'll end up with a double slash either way.

**Review by mhevery — APPROVED** [review]
LGTM, please answers comments from others.

**Pull request closed without merging**